bin/live/pycbc_live_collate_triggers

```diff
@@ -0,0 +1,333 @@
+    if not args.trigger_dir:
+        parser.error('Please provide a directory containing trigger files.')
+
+    # Convert dates to datetime to get the days array
+    start_date = datetime.strptime(args.start_date, '%Y-%m-%d').date()
+    num_days = timedelta(days=args.num_days - 1)
+    end_date = start_date + num_days
+
+    # Create the days array
+    days = []
+    delta = timedelta(days=1)
+
+    # Loop through the days and add each date to the list
+    current_date = start_date
+    while current_date <= end_date:
+        days.append(current_date)
+        current_date += delta
+
+    # Convert to strings in a list
+    days = [str(date).replace('-', '_') for date in days]
+    num_days = str(len(days))
+
+    initial_trigger_files = [
+        os.path.join(args.trigger_dir, day, trigger_file)
+        for day in days
+        for trigger_file in os.listdir(os.path.join(args.trigger_dir, day))
+    ]
+
+if args.trigger_file_method == 'gps-start-end-time':
+    if not args.gps_start_time:
+        parser.error('Please provide a GPS start time.')
+    if not args.gps_end_time:
+        parser.error('Please provide a GPS end time.')
+    if not args.trigger_dir:
+        parser.error('Please provide a directory containing trigger files.')
+
+    initial_trigger_files = [
+        l for l in glob.glob(args.trigger_dir + '/*/*-Live-*.hdf', recursive=True)
+        if float(l.split('/')[-1].split("-")[2]) > args.gps_start_time and float(l.split('/')[-1].split("-")[2]) < args.gps_end_time
+    ]
+
+logging.info(f" {len(initial_trigger_files)} files found")
+
+# Check if all files found are in the correct format and are trigger files
+ifo_permutations = [''.join(p) for p in itertools.permutations(args.ifos)]
+ifo_prefix_set = set(ifo_permutations)
+
+trigger_files = []
+for source_file in initial_trigger_files:
+    trigger_file = os.path.basename(source_file)
+    if any(trigger_file.startswith(prefix + '-Live') for prefix in ifo_prefix_set):
+        trigger_files.append(source_file)
+
+logging.info(f" {len(trigger_files)} confirmed trigger files")
+
+if args.output_trigger_file_list:
+    logging.info(' Writing list of trigger files to: ' + args.output_trigger_file_list)
+    with open(args.output_trigger_file_list, 'w') as f:
+        for item in trigger_files:
+            f.write("%s\n" % item)
+
+###########################
+# COLLATE THE TRIGGER FILES
+###########################
+
+if args.output_file_name:
+    output_file = args.output_dir + args.output_file_name
+else:
+    # Extract the gpstimes for every file and take min/max for start and end
+    trigger_file_times = [float(trigger_file.split('/')[-1].split("-")[2])
+                          for trigger_file in trigger_files]
+    start_gpstime = min(trigger_file_times)
+    end_gpstime = max(trigger_file_times)
+
+    ifo_string = "".join(args.ifos)
+    output_file = args.output_dir + f'{ifo_string}-Live-' + start_gpstime + '-' + end_gpstime + '.hdf'
```

Comment:
```suggestion
    output_file = f'{args.output_dir}/{ifo_string}-Live-{start_gpstime}-{end_gpstime}.hdf'
```
…but note that the file naming convention calls for the last integer value to be the *duration* of the data (rounded up to the next integer second), not the end time.